iquip/apps/scheduler.py

```diff
@@ -34,7 +34,7 @@ def _dismiss_items(layout: Optional[QLayout] = None):
                 _dismiss_items(item.layout())
 
 
-def _run_thread_with_worker(worker: QObject, parent: Optional[QObject] = None):
+def _thread_with_worker(worker: QObject, parent: Optional[QObject] = None):
     """Runs another thread with given worker.
```

Comment:
Good, now please annotate the return type: `def _thread_with_worker(...) -> QThread:`.
Also, please update the docstring summary as well: `"""Returns a new thread with given worker.`